components/phone_com/app/phone_com.app.ts

```diff
@@ -1,13 +1,71 @@
 import { defineApp } from "@pipedream/types";
+import axios from "axios";
+import { httpRequestParams, apiResponse, sendMessageParams } from "../common/types";
 
 export default defineApp({
   type: "app",
   app: "phone_com",
-  propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _baseUrl(): string {
+      return 'https://api.phone.com/v4';
     },
+    async _httpRequest({ method, endpoint, data }: httpRequestParams): apiResponse {
+      return axios({
+        method,
+        url: this._baseUrl() + endpoint,
+        headers: {
+          Authorization: `Bearer ${this.$auth.access_token}`,
+        },
+        data
+      })
+    },
```

Comment:
We should pass the `$` variable into `_httpRequest` to be used with Pipedream axios.
```suggestion
    async _httpRequest({ method, endpoint, data }: httpRequestParams, $ = this): apiResponse {
      return axios($, {
        method,
        url: this._baseUrl() + endpoint,
        headers: {
          Authorization: `Bearer ${this.$auth.access_token}`,
        },
        data
      })
    },
```